### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by alexmyczko** [vermaseren/form#404]
Is there a plan to tag a new release? Ubuntu 22.04 LTS will release soon, and their archive sync deadline from Debian is around mid of Feb 2022...

**Comment by tueda** [comment]
Indeed, I think it is a good idea to consider a patch release 4.2.2 from the current code, if others are fine.

**Comment by alexmyczko** [comment]
what does it look like?

**Comment by benruijl** [comment]
I want to release a new version this month. @tueda are there any issues that need to be resolved before the new release?

**Comment by vermaseren** [comment]
I have a number of things that should go in this new version.
Unfortunately, on my new computer I did not get the git to go properly.
I will try to solve that the coming few days. If I cannot I may need some help.
It did not want to make proper connection with the github.

Jos

**Comment by tueda** [comment]
@benruijl At least, some easy stuff can be resolved, like #389. I think #400 is almost(?) resolved.

We may also use this occasion to enable/start [GitHub Discussions](https://docs.github.com/en/discussions), which would be the place where people can ask any questions more frankly (with the "Q & A" category). For this, the bottom lines in README should be updated.

**Comment by tueda** [comment]
I will fix #389: the last line of p.211 of [the manual](https://github.com/vermaseren/form/suites/9070472098/artifacts/419770370) should be
```
-/+d_(mu,mu)*g_(1,m1,m2,...,mj,mn)
```
instead of
```
-/+2*d_(mu,mu)*g_(1,m1,m2,...,mj,mn)
```

**Comment by tueda** [comment]
We have released v4.3.0. This is a minor release with some additional features and bug fixes. In this sense, it might be appropriate to call it version 4.2.2. But 3 years and 9 months since the previous release are long enough to make us [sentimental](http://sentimentalversioning.org/), so 4.3.0. :-)
